modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -87,3 +87,73 @@ $ podman run --pull=always --rm --volume $HOME/clusterconfig:/pwd --workdir /pwd
 This command produces a combined Ignition config in `$HOME/clusterconfig/worker-raid.ign`.
 +
 . Continue with the remainder of the {product-title} deployment, using the combined Ignition config to provision nodes.
+
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume.
+
+. Create a machine config that configures a data volume by using software RAID.
+
+* To configure a data volume with RAID 1 on secondary disks, save the following machine config sample to a file in the `<installation_directory>/openshift` directory. For example, name the file `raid1-alt-storage.yaml`:
++
+.RAID 1 on secondary disks
+[source, yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: raid1-alt-storage
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+      - device: /dev/sdc
+        partitions:
+        - label: data-1
+        wipeTable: true
+      - device: /dev/sdd
+        partitions:
+        - label: data-2
+        wipeTable: true
+      filesystems:
+      - device: /dev/md/md-data
+        format: xfs
+        path: /var/lib/containers
+        wipeFilesystem: true
+      raid:
+      - devices:
+        - /dev/disk/by-partlabel/data-1
+        - /dev/disk/by-partlabel/data-2
+        level: raid1
+        name: md-data
+    systemd:
+      units:
+      - contents: |-
+          [Unit]
+          Before=local-fs.target
+          Requires=systemd-fsck@dev-md-md\x2ddata.service
+          After=systemd-fsck@dev-md-md\x2ddata.service
+
+          [Mount]
+          Where=/var/lib/containers
+          What=/dev/md/md-data
+          Type=xfs
+
+          [Install]
+          RequiredBy=local-fs.target
+        enabled: true
+        name: var-lib-containers.mount <1>
+----
++
+<1> If you choose a different mount point, you must update the unit name to correspond to your mount point. Otherwise the unit will not activate. You can generate a matching unit name with `echo $(systemd-escape -p $mountpoint).mount` where `$mountpoint` is your chosen mount point.
++
+. Apply the config to your cluster by running the following command:
++
+[source, terminal]
+----
+$ oc create -f raid1-alt-storage.yaml
```

Comment:
You might want to replace `raid1-alt-storage.yaml` with a user-replaceable variable such as `<raid_config_file>`. Sometimes users just copy and run code snippets without changing things, so it's best to make it super clear it's an example. If you want to use your example from above, you could put something like this under it:
> where `<raid_config_file>` is your RAID configuration file, for example `raid1-alt-storage.yaml`.